src/makeCalculator.js

```diff
@@ -4,7 +4,37 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    operate(callback, a) {
+      this.result = callback(this.result, a);
```

Comment:
```suggestion
    operate(callback, value) {
      callback.call(result, value);
```

Reply:
After this change my code working wrong. Can i keep my option?